stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -2508,12 +2489,11 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     //
     
     /// The hash value for the data.
-    @inlinable
-    public var hashValue: Int {
-        return _representation.hashValue
+    @inline(never) // This is not inlinable as emission into clients could cause cross-module inconsistencies if they are not all recompiled together.
```

Comment:
Now that this isn't inlinable (👍🏻) you should be able to safely remove the inalienability from all the representation.hashable implementations.